app/models.py

```diff
@@ -328,10 +328,13 @@ class SupplierFramework(db.Model):
                              primary_key=True)
     declaration = db.Column(JSON)
     on_framework = db.Column(db.Boolean, nullable=True)
+    agreed_variations = db.Column(JSON)
+
+    # The following three fields are deprecated as this information now lives in the
```

Comment:
Maybe word this comment more strongly - e.g. `The following three fields are deprecated, no longer updated, may contain outdated data and MUST NOT BE USED. The framework_agreements table is now the only the source of truth for this data.`